agent/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "agent"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
```

It would be best to add a license line here.
```suggestion
license = "Apache-2.0 OR MIT"

```